packages/design-system/src/styles/components/_UsaBanner.scss

```diff
@@ -1,13 +1,15 @@
 $usa-banner-background-color: #f0f0f0;
 $usa-banner-text-color: #1b1b1b;
+$usa-banner-close-image-background-color: #e6e6e6;
+$usa-banner-header-action-color: #005ea2;
 
 .ds-c-usa-banner {
   background-color: $usa-banner-background-color;
   color: $usa-banner-text-color;
   font-family: 'Source Sans Pro Web', 'Helvetica Neue', Helvetica, Roboto, Arial, sans-serif;
```

Comment:
This isnt a font family we currently provide right? Again, this code is coming from compiled CSS. We should be defining and using SCSS variables and working with the CMSDS CSS architecture